app/src/main/java/org/kiwix/kiwixmobile/local_file_transfer/ReceiverDeviceAsyncTask.java

```diff
@@ -106,10 +106,10 @@ protected Boolean doInBackground(Void... voids) {
   @Override
   protected void onProgressUpdate(Integer... values) {
```

Comment:
change the type to of this class from `AsyncTask<Void, Integer, Boolean>` to `AsyncTask<Void, FileStatus, Boolean>` and then you can publish progress with the Enum directly

Reply:
That could cause errors as on progress Update we need to pass the fileIndex which is an Integer.

Reply:
ah so it is